wazuh-dashboard/config/wazuh.yml

```diff
@@ -0,0 +1,7 @@
+hosts:
+  - default:
+     url: https://wazuh.manager
+     port: 55000
+     username: wazuh-wui
+     password: wazuh-wui
+     run_as: false
```

Comment:
Why don't you enable run_as?

Reply:
use the configuration that existed in the file [wazuh_app_config.sh](https://github.com/wazuh/wazuh-docker/blob/master/kibana-odfe/config/wazuh_app_config.sh)